gulpfile.js

```diff
@@ -225,7 +225,8 @@ gulp.task('enforce-format', function() {
 function doCheckFormat() {
   return gulp
       .src([
-        'gulpfile.js', 'karma-test-shim.js', 'misc/api-doc.js', 'misc/api-doc.spec.js', 'misc/demo-gen.js', PATHS.src
+        'gulpfile.js', 'karma-test-shim.js', 'misc/api-doc.js', 'misc/api-doc.spec.js', 'misc/demo-gen.js', PATHS.src,
+        '!src/util/keys.ts'
```

Comment:
I don't agree there should be exceptions for code formatting in the library source code at the file level

Reply:
I don't agree there should be code formatting automatic enforcement at all given its poor results.

I've let it until we have to integrate this PR, in case I have to change things, because honestly, working on the output of the formatter is a pain in the ass, to remain polite.